apps/studio/src/features/editing-experience/components/ComplexEditorStateDrawer.tsx

```diff
@@ -29,7 +29,7 @@ export default function ComplexEditorStateDrawer(): JSX.Element {
   const { title } = getComponentSchema(component.type)
 
   return (
-    <Box position="relative" h="100%" w="100%">
+    <Box position="relative" h="100%" w="100%" overflow="scroll">
```

Comment:
nit: prefer `overflow="auto"` instead of scroll, if not scrollbar will forever be shown even if nothing is overflowing

Reply:
Updated in 7f2ee305b59a98499427b90037631637e6b43895.